apps/condo/domains/division/hooks/useTableColumns.tsx

```diff
@@ -0,0 +1,96 @@
+import { colors } from '@condo/domains/common/constants/style'
+import { ColumnsType, FilterValue } from 'antd/es/table/interface'
+
+import React, { useMemo } from 'react'
+import { useIntl } from '@core/next/intl'
+
+// import { createSorterMap, IFilters } from '../utils/helpers'
+import get from 'lodash/get'
+
+import { Typography } from 'antd'
+import { isEmpty } from 'lodash'
+import { Highliter } from '@condo/domains/common/components/Highliter'
+import { getTextFilterDropdown, getFilterIcon } from '@condo/domains/common/components/TableFilter'
+import { EmptyTableCell } from '@condo/domains/common/components/EmptyTableCell'
+
+export interface ITableColumn {
+    title: string,
+    ellipsis?: boolean,
+    sortOrder?: string,
+    filteredValue?: FilterValue,
+    dataIndex?: string,
+    key?: string,
+    sorter?: boolean,
+    width?: string,
+    filterDropdown?: unknown,
+    filterIcon?: unknown
+}
+
+const getFilteredValue = (filters: any, key: string | Array<string>): FilterValue => get(filters, key, null)
+
+export const useTableColumns = (sort: Array<string>, filters: any,
+    setFiltersApplied: React.Dispatch<React.SetStateAction<boolean>>): Array<ITableColumn> => {
+    const intl = useIntl()
+    const columns = useMemo(() => {
+        const DivisionTitleMessage = intl.formatMessage({ id: 'pages.condo.property.index.TableField.Division' })
+        const BuildingsTitleMessage = intl.formatMessage({ id: 'pages.condo.property.index.TableField.Buildings' })
+        const ForemanTitleMessage = intl.formatMessage({ id: 'pages.condo.property.index.TableField.Foreman' })
+        const TechiesTitleMessage = intl.formatMessage({ id: 'pages.condo.property.index.TableField.Techies' })
+        // const sorterMap = createSorterMap(sort)
+        const search = getFilteredValue(filters, 'search')
+        const render = (text, isArray = false) => {
```

Comment:
Use common render as soon as https://github.com/open-condo-software/condo/pull/498 will go to master
Will be placed in `@condo/domains/common/components/Table/Renders`